Makefile

```diff
@@ -10,6 +10,7 @@ GOTOOLS = \
 	github.com/golang/dep/cmd/dep \
 	github.com/alecthomas/gometalinter \
 	github.com/rakyll/statik
+GOBIN ?= $(GOPATH)/bin
```

Comment:
Don't we want users and systems to have a properly set Golang environment?

Reply:
That is a conditional assignment reuqired by `get_dev_tools`

Reply:
`get_dev_tools` requires `GOBIN` to be set. I'm saying this should already be set properly, no?

Reply:
Not necessarily. Go relies on either `$GOBIN` or `$GOPATH`/bin

Reply:
I think it's safe to assume that most users don't have GOBIN set as it's not strictly required.

Reply:
`$GOBIN` should always be set. `$GOPATH/bin` is typically the location for `$GOBIN`. I suppose this is fine, but maybe it'll introduce some space for user error.

Reply:
> $GOBIN should always be set.

I disagree - e.g. it does not play nicely with cross-compilation for instance. That variable though is set if and only if the user didn't set it beforand. Thus I don't foresee a problem honestly.